qiime2/core/testing/plugin.py

```diff
@@ -152,6 +157,24 @@
 )
 
 dummy_plugin.methods.register_function(
+    function=long_description_method,
+    inputs={},
+    parameters={
+        'name': qiime2.plugin.Str,
+        'age': qiime2.plugin.Int
+    },
+    parameter_descriptions={
+        'name': """This is a very long description. If asked about its length,\
+ I would have to say it is greater than 79 characters."""
+    },
+    outputs=[
+        ('out', Mapping)
+    ],
+    name="Long Description",
+    description="This method does nothing, but includes a long annotation"
```

Comment:
Make this description wrap as well (you can use lorem ipsum if that helps). Also add input and output descriptions. It's kind of a waste of a good method otherwise.